docker/gogs/app.ini

```diff
@@ -0,0 +1,54 @@
+BRAND_NAME = Conduit Suite Git Repos
+RUN_USER   = git
+RUN_MODE   = prod
```

Comment:
What are the alternatives, and implications of `prod` here?

Reply:
I'm not sure tbh. This was the default in the shipped configuration file for Gogs, and it worked, so I didn't look any further. I assume it would enable more visibility into Gogs internals for development purposes, but as we are using it as a drop-in appliance, I don't think we need it.